src/frontend/JasmineGraphFrontEnd.cpp

```diff
@@ -576,3 +611,513 @@ void JasmineGraphFrontEnd::removeGraph(std::string graphID, void *dummyPt) {
     sqlite->runUpdate("DELETE FROM graph WHERE idgraph = " + graphID);
 }
 
+
+long JasmineGraphFrontEnd::countTriangles(std::string graphId, void *dummyPt) {
+    long result= 0;
+    Utils utils;
+    vector<std::string> hostList = utils.getHostList();
+    int hostListSize = hostList.size();
+    int counter = 0;
+    std::vector<std::future<long>> intermRes;
+    map<std::string,std::future<std::string>> remoteCopyRes;
+    PlacesToNodeMapper placesToNodeMapper;
+
+    string sqlStatement = "SELECT name,partition_idpartition FROM host_has_partition INNER JOIN host ON host_idhost=idhost WHERE partition_graph_idgraph=" + graphId + ";";
+
+    SQLiteDBInterface *sqlite = (SQLiteDBInterface *) dummyPt;
+    std::vector<vector<pair<string, string>>> results = sqlite->runSelect(sqlStatement);
+
+    std::map<string, std::vector<string>> map;
+
+    // Implement Logic to Decide the Worker node which Acts as the centralstore triangle count aggregator
+    std::string aggregatorWorker = "localhost@7780";
```

Comment:
Why is this value hard coded? Why is it localhost? Why do we use the port 7780 here rather than getting it from the constants?

Reply:
That is because the central store aggregator worker deciding logic yet to be implemented depending on the worker load and available resources. So till it is implemeted I kept it hard coded